tests/console/mdadm.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright © 2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: mdadm test, run script creating RAID 0, 1, 5, re-assembling and replacing faulty drive
+# Maintainer: Jozef Pupava <jpupava@suse.com>
+
+use base 'consoletest';
+use testapi;
+use strict;
+
+sub run {
+    select_console 'root-console';
+
+    assert_script_run 'wget ' . data_url('qam/mdadm.sh');
```

Comment:
This bash script is enormous, you did a good job putting all comments during the execution, but I still a bit concerned how easy will it be to debug failures in that script

Reply:
the script wrote dabatianni, I just did small modification
It will be super easy because you can take the script and run in anywhere e.g. with -x

Reply:
Hmm, actually good point, what's about adding it here? I believe it can be pretty useful

Reply:
I would leave it without it, I think the script is simple (I did not even write it :sunglasses:), I will run it locally to debug